src/main/java/com/aws/greengrass/mqttbridge/MQTTBridge.java

```diff
@@ -148,15 +148,18 @@ public void startup() {
             mqttClient = new MQTTClient(brokerUri, clientId, mqttClientKeyStore, executorService);
             mqttClient.start();
             messageBridge.addOrReplaceMessageClient(TopicMapping.TopicType.LocalMqtt, mqttClient);
+            messageBridge.updateSubscriptionsForClient(TopicMapping.TopicType.LocalMqtt, mqttClient);
```

Comment:
if these two calls _always_ go together, why not just update the subscriptions when replacing the client?

Reply:
sounds good, modified method to `addOrReplaceMessageClientAndUpdateSubscription()`, clubbing those two.